importer/importer_utils.py

```diff
@@ -324,8 +324,11 @@ def get_text_inside_brackets(text):
             raise ValueError("Unmatched brackets encountered.")
         last_bracket_index += first_bracket_index
         result = text[first_bracket_index + 1:last_bracket_index]
-        if result.find(')') > 0 and result.find('(') > result.find(')'):
-            raise ValueError("Unmatched brackets encountered.")
+        if "(" in result or ")" in result:
```

Comment:
You can drop this conditional (since the next was augmented it already covers this as well (essentially the `result.find(')') > 0` and `result.find('(') > 0`).